.github/workflows/release.yml

```diff
@@ -50,18 +53,20 @@ jobs:
 
       - name: Maven release ${{steps.metadata.outputs.current-version}}
         run: |
-          gpg --quiet --batch --yes --decrypt --passphrase="${{secrets.GPG_PASSPHRASE}}" --output /tmp/maven-settings.xml .github/release/maven-settings.xml.gpg
           git checkout -b release
-          mvn -B release:prepare -Prelease -DreleaseVersion=${{steps.metadata.outputs.current-version}} -DdevelopmentVersion=${{steps.metadata.outputs.next-version}} -s /tmp/maven-settings.xml
+          ./mvnw -B release:prepare -Prelease -DreleaseVersion=${{steps.metadata.outputs.current-version}} -DdevelopmentVersion=${{steps.metadata.outputs.next-version}}
           git commit -am "Update stable version for documentation"
           git checkout ${{github.base_ref}}
           git rebase release
-          mvn -B release:perform -Darguments=-DperformRelease -DperformRelease -Prelease -s /tmp/maven-settings.xml
+          ./mvnw -B release:perform -Darguments=-DperformRelease -DperformRelease -Prelease
+        env:
+          MAVEN_USERNAME: ${{ secrets.OSSRH_USERNAME }}
+          MAVEN_PASSWORD: ${{ secrets.OSSRH_TOKEN }}
```

Comment:
Are you sure we are ready for that? We would need to define the secrets and I'm not sure we have them defined already?

Reply:
@gastaldi can you confirm everything is set up on the project ?

Reply:
@gastaldi ah you already prepared things for this? If it's the case, I missed the info.

Reply:
@gsmet yeah, that is the default since Dec 2021. See what's generated with `quarkus create extension amazon-services`

Reply:
@scrocquesel the release script has evolved since the original PR was issued. I created https://github.com/quarkiverse/quarkus-amazon-services/pull/679 to have those changes incorporated here

Reply:
Thanks, I merged the PR and backport it here